package.json

```diff
@@ -76,7 +76,7 @@
     "ncp": "^2.0.0",
     "object-assign": "^4.1.1",
     "platform": "^1.1.0",
-    "prettier": "^1.2.2",
+    "prettier": "1.2.2",
```

Comment:
Perhaps? But that won't do anything for folks who are using npm (like me), and [the official contribution doc](https://facebook.github.io/react/contributing/how-to-contribute.html) has its instructions in npm, so that's what I thought was supported.